base/Base.jl

```diff
@@ -186,9 +186,10 @@ end
 include(strcat((length(Core.ARGS)>=2 ? Core.ARGS[2] : ""), "build_h.jl"))     # include($BUILDROOT/base/build_h.jl)
 include(strcat((length(Core.ARGS)>=2 ? Core.ARGS[2] : ""), "version_git.jl")) # include($BUILDROOT/base/version_git.jl)
 
-# These used to be in build_h.jl and are retained for backwards compatibility
-const libblas_name = "libblastrampoline"
-const liblapack_name = "libblastrampoline"
+# These used to be in build_h.jl and are retained for backwards compatibility.
+# NOTE: keep in sync with `libblastrampoline_jll.libblastrampoline`.
+const libblas_name = "libblastrampoline" * (Sys.iswindows() ? "-5" : "")
+const liblapack_name = libblas_name
```

Comment:
Can we somehow mark this as deprecated and instead tell this [list of packages](https://juliahub.com/ui/Search?q=libblas_name&type=code) that they should use `libblastrampoline_jll.libblastrampoline` instead?

Reply:
Can you deprecate a variable (that is, with a nice warning when you try to access it)?  And I'm not sure that's a future-proof option, in case we come up with yet another blas idea in the future, `Base.libblas_name` looks more agnostic.